omod-1.8/src/test/resources/create_patient_from_get.json

```diff
@@ -0,0 +1,267 @@
+                    }
+                ]
+            },
+            "location":{
+                "uuid":"8d6c993e-c2cc-11de-8d13-0010c6dffd0f",
+                "display":"Outpatient Clinic",
+                "links":[
+                    {
+                        "rel":"self",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/location/58c57d25-8d39-41ab-8422-108a0c277d98"
+                    }
+                ]
+            },
+            "preferred":true,
+            "voided":false,
+            "links":[
+                {
+                    "rel":"self",
+                    "uri":"http://192.168.1.100/openmrs/ws/rest/v1/patient/a65fc361-f6ac-40ba-8485-94749c061509/identifier/035be43e-a902-48bc-a22c-0ad01aaff731"
+                },
+                {
+                    "rel":"full",
+                    "uri":"http://192.168.1.100/openmrs/ws/rest/v1/patient/a65fc361-f6ac-40ba-8485-94749c061509/identifier/035be43e-a902-48bc-a22c-0ad01aaff731?v=full"
+                }
+            ],
+            "resourceVersion":"1.8"
+        }
+    ],
+    "person":{
+        "uuid":"a65fc361-f6ac-40ba-8485-94749c061509",
+        "display":"Jonnea Bijo",
+        "gender":"M",
+        "age":23,
+        "birthdate":"1995-01-01T00:00:00.000+0000",
+        "birthdateEstimated":true,
+        "dead":false,
+        "deathDate":null,
+        "causeOfDeath":null,
+        "preferredName":{
+            "display":"Jonnea Bijo",
+            "uuid":"0a6e7017-e6e7-42f0-ba99-c6d82eecb164",
+            "givenName":"Jonnea",
+            "middleName":null,
+            "familyName":"Bijo",
+            "familyName2":null,
+            "voided":false,
+            "links":[
+                {
+                    "rel":"self",
+                    "uri":"http://192.168.1.100/openmrs/ws/rest/v1/person/a65fc361-f6ac-40ba-8485-94749c061509/name/0a6e7017-e6e7-42f0-ba99-c6d82eecb164"
+                },
+                {
+                    "rel":"full",
+                    "uri":"http://192.168.1.100/openmrs/ws/rest/v1/person/a65fc361-f6ac-40ba-8485-94749c061509/name/0a6e7017-e6e7-42f0-ba99-c6d82eecb164?v=full"
+                }
+            ],
+            "resourceVersion":"1.8"
+        },
+        "preferredAddress":{
+            "display":"d",
+            "uuid":"ebd2ed47-6d13-41a0-bc6b-c1ce6b10da25",
+            "preferred":true,
+            "address1":"d",
+            "address2":null,
+            "cityVillage":null,
+            "stateProvince":null,
+            "country":null,
+            "postalCode":null,
+            "countyDistrict":null,
+            "address3":null,
+            "address4":null,
+            "address5":null,
+            "address6":null,
+            "startDate":null,
+            "endDate":null,
+            "latitude":null,
+            "longitude":null,
+            "voided":false,
+            "address7":null,
```

Comment:
Same here, these were added in 2.0 so it should not be here, it's one of the reasons why the test fails since this file is in the 1.8 subproject and the test where it is used is a 1.9 test